src/idea.js

```diff
@@ -0,0 +1,19 @@
+class Idea {
+    constructor(id, title, body) {
+        this.id = Date.now();
+        this.title = title;
+        this.body = body;
+        this.star = false;
```

Comment:
This is a minor suggestion but consider having a more descriptive name rather than `this.star` - I am guessing that this has to do with a favorited idea. If I were a new dev on the team looking through the code base, this may be slightly confusing. Maybe something like `this.isFavorited` or `this.isStarred`. It's good to think of booleans as always asking a yes or no question, so your naming can correlate to that.